apps/src/lib/kits/maker/PlaygroundComponents.js

```diff
@@ -226,6 +231,10 @@ function initializeAccelerometer(board) {
     board,
     controller: PlaygroundIO.Accelerometer
   });
+
+  accelerometer.start = function () {
+    accelerometer.io.sysexCommand([CP_COMMAND, CP_ACCEL_STREAM_ON]);
+  };
```

Comment:
Obviously this should move down into forked playground-io at some point (maybe all three extra methods here should) but I'd like to get this unbroken first.